packages/sp/files/types.ts

```diff
@@ -37,6 +37,15 @@ export class _Files extends _SharePointQueryableCollection<IFileInfo[]> {
         return tag.configure(File(this).concat(`('${escapeQueryStrValue(name)}')`), "fis.getByName");
     }
 
+    /**
+     * Gets a File by file absolute url
+     *
+     * @param fileUrl The absolute url of the file
+     */
+     public getUsingAbsoluteUrl(fileUrl: string): IFile {
+        return tag.configure(File(`${fileUrl}`), "fis.getFileUsingResponseUrl");
```

Comment:
Should this be escaped and put into an alias to prevent path length issues?

Reply:
I actually haven't encountered that issue before so didn't consider that. Reading up on it now and it seems to make sense. I would update the PR.